craft_application/fetch.py

```diff
@@ -294,6 +289,28 @@ def configure_instance(
     return net_info.env
 
 
+def get_log_filepath() -> pathlib.Path:
+    """Get the path containing the fetch-service's output."""
+    # All craft tools log to the same place, because it's a single fetch-service
+    # instance. It needs to be a location that the fetch-service, as a strict
+    # snap, can write to.
+    logdir = _get_service_base_dir() / "craft-logs"
+    logdir.mkdir(exist_ok=True, parents=True)
+    return logdir / "fetch-service.txt"
```

Comment:
Do we want to follow our craft-naming there `*craft-<timestamp>.log`? 
I'd at least suggest keeping `.log` suffix as `.txt` does not usually indicate logfile 
```suggestion
    return logdir / "fetch-service.log"
```

Reply:
Using `craft-<timestamp>.log` is not possible now because the fetch-service instance (and its log) is shared between running applications, and it's useful that even an application that uses an already-running fetch-service can print out the path to the log. Once/if we move to a per-application strategy then yeap

I did rename it to `.log`, though. Thanks!